new_listings_scraper.py

```diff
@@ -66,11 +67,13 @@ def get_last_coin():
 
     uppers = None
 
-    if 'Will List' not in latest_announcement or found_coin[0] == existing_coin:
+    if 'Will List' not in latest_announcement or found_coin[0] == existing_coin or \
```

Comment:
Why is there a `\` here?  is that only because it wraps the if statement that way??